ccd-definition/CaseEventToComplexTypes/create-order/create-order-submitted.json

```diff
@@ -5,18 +5,28 @@
     "CaseEventID": "createOrder",
     "CaseFieldID": "orderTypeAndDocument",
     "ListElementCode": "type",
-    "EventElementLabel": "Type of order",
```

Comment:
reason for change: header on the 'Type of order' page was actually incorrect - now properly displays the question 'Which type of order are you making?'

Reply:
can move to next story if desired (slightly out of scope?), as there is an upcoming story which adds a new page for interim orders